### DISCUSSION TITLE
Colorized manage.py test's results in docker container

### DISCUSSION THREAD

**Discussion by huydbui** [cookiecutter/cookiecutter-django#3570] [Q&A]
Most of my time spending on a project is in the django docker container running 'python manage.py test'
I'm wondering if there is an easy way to enable the colored output, green/red thing. I have tried https://pypi.org/project/redgreenunittest/ but it doesn't seem to work for me.

Did I miss something simple, could someone please let me know?

**Comment by foarsitter** [comment]
First thing to notice: this project does not use the django testrunner but relies on pytest. Tests are executed with the `pytest` command, as demonstrated in [gitlab-ci.yml](https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/.gitlab-ci.yml)

Can you try  that route?

**Comment by huydbui** [comment]
Yes, I do see some color in pytest which is good. But I don't use pytest because of this:

```
root@2f92e0b6d8d5:/app# pytest
Test session starts (platform: linux, Python 3.9.9, pytest 6.2.5, pytest-sugar 0.9.4)
django: settings: config.settings.test (from option)
rootdir: /app, configfile: pytest.ini
plugins: Faker-12.0.0, django-4.5.2, sugar-0.9.4
collecting ...
```

The "collecting ..." part takes at least 2 minutes every time I run "pytest". Is that normal or am I doing something wrong? Thank you for your response.

**Comment by huydbui** [comment]
Oh, thank you for your note. I did pytest this time for a specific package and it works fast (i didn't test on individual package before). So the pytest on the suite takes 2 minutes and the python manage.py test takes <10s is the only thing I have left. If it's normal, please let me know.

**Reply by foarsitter** [reply]
I'm running 453 testes in 30 seconds or so in a Docker environment. Do you have a configuration for pytest in [pytest.ini](https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/pytest.ini)? Since the django testrunner is super fast your machine is not the problem. I suppose your configuration has an issue. 

Try a clean project of this template and running the tests will give you some insights.
